API/SSW.Rewards.Application/Quizzes/Queries/GetQuizDetailsQuery/AutoMapperProfile.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AutoMapper;
+
+namespace SSW.Rewards.Application.Quizzes.Queries.GetQuizDetailsQuery
+{
+    public class AutoMapperProfile : Profile
+    {
+        public AutoMapperProfile()
+        {
+            CreateMap<Domain.Entities.Quiz, QuizDetailsDto>()
+                .ForMember(dst => dst.QuizId, opt => opt.MapFrom(src => src.Id));
+            //.ForMember(dst => dst.Questions, opt => opt.MapFrom(src => src.Questions));
```

Comment:
Remove the unused lines from these profiles

Reply:
Done